ios/Classes/SwiftFlutterPlugin.swift

```diff
@@ -154,7 +158,32 @@ public class SwiftFlutterPlugin: NSObject, Flutter.FlutterPlugin {
         group.wait()
 
     }
-
+    
+    // MARK: WRITE FILE
```

Comment:
Is this comment still needed?

Reply:
It's more or less a convenience comment used in Xcode projects. A comment with MARK is seen in an Xcode file's sidebar (that shows the zoomed out view of the entire file) as large print. So you can more easily navigate to each method at a glance